### DISCUSSION TITLE
Enterprise Key Words

### DISCUSSION THREAD

**Discussion by nthoman** [pnp/syntex-samples#5] [Ideas]
Once a document is classified, it would be great if we could also hardcode some Enterprise Key Words to a document. Currently I do this this with a "Lookup List" and a Power Automate workflow that is triggered after a document is classified.

**Comment by jameseccles** [comment]
Could you expand on this to make sure I understand?

Do you mean that if a file is classified by Syntex, you always want the same X number of keywords applied to every file?
